client/EditContributionView.js

```diff
@@ -59,15 +60,42 @@ const styles = theme => ({
     previewButton: {
         display: 'block',
         position: 'fixed',
+        height: 60,
         bottom: 20,
-        right: 20,
+        right: 10,
+    },
+    approvalPaper: {
+        height: 95,
+        position: 'fixed',
+        width: 100,
+        bottom: 95,
+        left: 10,
+    },
+    publishPaper: {
+        height: 65,
+        position: 'fixed',
+        bottom: 20,
+        width: 100,
+        left: 10,
     },
     reviewOptionLeft: {
         marginLeft: '10%',
     },
     reviewOptionRight: {
         marginRight: '10%',
     },
+    cardColor: {
+        backgroundColor: '#fce4ec',
+    },
+    paper: {
+        ...theme.mixins.gutters(),
```

Comment:
is this mixin still required in the latest version of material-ui?

Reply:
the page looks different when I take it out, so I'll leave it here for now